build.gradle.kts

```diff
@@ -53,6 +53,7 @@ repositories {
 dependencies {
     implementation(kotlin("stdlib-jdk8"))
     implementation(kotlin("reflect"))
+    implementation("com.google.guava:guava:30.1-android")
```

Comment:
Android flavour for JRE and Android [compatibility](https://github.com/google/guava/wiki/ReleasePolicy#flavors).

Reply:
Hopefully it will soon be 🎶 time to say goodbye 
😄